test/karma.config.ts

```diff
@@ -57,13 +58,24 @@ export function config(config) {
     customLaunchers: customLaunchers,
 
     exclude: [],
-    preprocessors: {},
-    reporters: ['dots'],
+    preprocessors: {
+      'dist/**/!(*spec).js': ['coverage']
+    }
+    reporters: ['dots', 'coverage'],
```

Comment:
I _think_ we force `--reporters=dots` on the command line in `build-and-test.sh`.